ICLabel.py

```diff
@@ -173,35 +177,66 @@ def run_iclabel(images: np.ndarray, psds: np.ndarray, autocorr: np.ndarray) -> n
     return labels.detach().numpy()
 
 
-def mne_to_eeglab_locs(raw):
+def mne_to_eeglab_locs(raw: mne.io.BaseRaw):
+    """Obtain EEGLab-like spherical coordinate from EEG channel positions.
+
+    TODO: @JACOB:
+    - Where is (0,0,0) defined in MNE vs EEGLab?
+    - some text description of how the sphere coordinates differ between MNE
+    and EEGLab.
+
+    Parameters
+    ----------
+    raw : mne.io.BaseRaw
+        Instance of raw object with a `mne.montage.DigMontage` set with
+        ``n_channels`` channel positions.
+
+    Returns
+    -------
+    Rd : np.array of shape (1, n_channels)
+        Angle in spherical coordinates of each EEG channel.
+    Th : np.array of shape (1, n_channels)
+        Degree in spherical coordinates of each EEG channel.
+    """
     def sph2topo(theta, phi):
+        """Add docstring in numpy format."""
         az = phi
         horiz = theta
         angle = -1 * horiz
         radius = (np.pi / 2 - az) / np.pi
         return angle, radius
 
     def cart2sph(x, y, z):
+        """Add docstring in numpy format."""
         azimuth = np.arctan2(y, x)
         elevation = np.arctan2(z, np.sqrt(x ** 2 + y ** 2))
         r = np.sqrt(x ** 2 + y ** 2 + z ** 2)
         # theta,phi,r
         return azimuth, elevation, r
 
-    locs = raw._get_channel_positions()
+    # get the channel position dictionary
+    montage = raw.get_montage()
+    positions = montage.get_positions()
+    ch_pos = positions['ch_pos']
 
-    # %% Obtain carthesian coordinates
+    # get locations as a 2D array
+    locs = np.vstack(ch_pos.values())
+
+    # Obtain carthesian coordinates
     X = locs[:, 1]
```

Comment:
One question I had was why is X in position 1, but Y is in position 0?

Perhaps it's the orientation of MNE vs EEGLab? If so, it'll be useful to document what's going on, so another dev can follow.